nmi_mysql/nmi_mysql.py

```diff
@@ -134,6 +149,8 @@ def _execute(self, _query, _params, _multi=False):
                         'affected_rows': result.rowcount
                     }
 
+            con.close()
```

Comment:
We really don't need close unless the pool is explicitly closed. All connection would be handled by the pool. (and close when the set active period is reached.